tools/build-appimage.sh

```diff
@@ -3,7 +3,7 @@
 # prepare folder
 mkdir -p ./AppDir/usr/bin ./AppDir/usr/share/icons
 cp -a ./src/. ./AppDir/usr/bin/.
-cp ./src/GUI/images/p2mm-icon.png ./AppDir/usr/share/icons/.
+cp ./src/GUI/images/p2mm-icon.ico ./AppDir/usr/share/icons/p2mm-icon.png # ICON MUST BE A PNG HENCE THE WEIRD COPY
```

Comment:
how does this beven work??
if you need a png why not copy a png file / convert the ico to png (ffmpeg is available on all distros) because this line only changes the name of the file not it's type